pkg/interface/src/views/apps/publish/components/NotePreview.tsx

```diff
@@ -62,8 +62,8 @@ export function NotePreview(props: NotePreviewProps) {
   return (
     <Link to={url}>
       <Col mb={4}>
-        <WrappedBox mb={1}>{title}</WrappedBox>
-        <WrappedBox mb={1}>
+        <WrappedBox mb={1}><Text bold>{title}</Text></WrappedBox>
+        <WrappedBox mb={1} className="md">
```

Comment:
I feel like we should defer to indigo-react's line-heights here instead of PRing more custom css classes?

Reply:
I agree with you, but `md` is the class applied to the contents of the note once you open it.